src/applet.c

```diff
@@ -227,8 +231,16 @@ static void brisk_menu_applet_init(BriskMenuApplet *self)
         gtk_widget_set_margin_end(image, 4);
         gtk_widget_set_halign(image, GTK_ALIGN_START);
 
+        /* Set the label text, if empty, show the default text */
+        gchar *text = g_settings_get_string(self->settings, "label-text");
+        if (g_str_equal(text, "")) {
+                self->label_text = _("Menu");
+        } else {
+                self->label_text = text;
+        }
+
         /* Now add the label */
-        label = gtk_label_new(_("Menu"));
+        label = gtk_label_new(self->label_text);
```

Comment:
GtkLabel will already dupe the string, so your floating reference has now disappeared, and leaked